app/views/transactions/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<header class="categories-header">
+    <%= render "shared/hamburger"%>
+    <h2 class="big-title">DETAILS</h2>
+    <%= link_to "Logout", destroy_user_session_path, data: { turbo_method: :delete }, class: 'logout' %>
+</header>
+
+<div class="trans action-wrap">
+  <div class="Image-name-container">
+   <div class="icon-container">
+     <img src="<%=@category.icon %>" alt="" class="category-icon">
+   </div>
+   <h2 class="category-name tranz"><%=@entity.name %></h2>
+  </div>
```

Comment:
- [ ] Kindly add a back button so users can easily navigate to the home page, this will create a better user experience